asf_search/search/search.py

```diff
@@ -154,21 +156,6 @@ def search(
     for key in join_fields:
         if key in data:
             data[key] = ','.join([str(v) for v in data[key]])
-    
-    # Special case to unravel WKT field a little for compatibility
-    if data.get('intersectsWith') is not None:
-        repaired_wkt = RepairWKT.repairWKT(data['intersectsWith'])
-        if "errors" in repaired_wkt:
-            raise ValueError(f"Error repairing wkt: {repaired_wkt['errors']}")
-        for repair in repaired_wkt["repairs"]:
-            warn(f"Modified shape: {repair}")
-        # DO we want unwrapped here??
-        opts.intersectsWith = repaired_wkt["wkt"]["wrapped"]
-        cmr_wkt = Input.parse_wkt_util(repaired_wkt["wkt"]["wrapped"])
-
-        (shapeType, shape) = cmr_wkt.split(':')
-        del data['intersectsWith']
-        data[shapeType] = shape
```

Comment:
Where was this moved to? The only way to save the repaired wkt to the returned opts, was to save it here (line 166).

```python
import asf_search
opts = asf_search.ASFSearchOptions(platform="S1", intersectsWith="GEOMETRYCOLLECTION(POINT(0 0), POINT(1 1))", maxResults=1)
results = asf_search.search(opts=opts)
results.searchOptions.intersectsWith
# Shows: 'GEOMETRYCOLLECTION (POINT (0.0000000000000000 0.0000000000000000), POINT (1.0000000000000000 1.0000000000000000))''GEOMETRYCOLLECTION (POINT (0.0000000000000000 0.0000000000000000), POINT (1.0000000000000000 1.0000000000000000))'
# Instead of something like: 'LINESTRING(0 0, 1 1)
```

Reply:
Fixed this, the new opts object gets assigned the validated wkt